package.json

```diff
@@ -23,6 +23,7 @@
     "lint:ci-report": "pnpm run lint:ci --format=junit --output-file=reports/junit/eslint-results.xml --max-warnings=0",
     "postinstall": "gulp fix-lockfile",
     "prepare": "husky install",
+    "svgo": "svgo packages/icons/*/*.svg",
```

Comment:
please install it in packages/scripts or packages/icons.

Reply:
No, need it to run in git hook. Installing in root directory would be better.

Reply:
I'm wondering, if you're running it in the source code, why you need to optimize it again in webpack?

Reply:
Formatting in git hook, minifying in building.

Reply:
The loader is not enabled in development mode.

Reply:
that's interesting